src/components/QuestionPlay/index.tsx

```diff
@@ -1,135 +1,156 @@
+        <Group>
+          <Title mr="xl" order={4}>
+            Question {selectedQuestion.questionNum}
+          </Title>
+          <Text weight="bold" component="span" size="sm">
+            Points: {points}
+          </Text>
+        </Group>
+        <Box my="xs" sx={getQuestionTextStyles}>
+          <Markdown>{text}</Markdown>
+        </Box>
+        {options.length === 1 ? (
+          // Show correct answer for question without options when answer is revealed
+          isAnswerRevealed ? (
+            <>
+              <Title mt="xl" order={6}>
+                Correct Answer
+              </Title>
+              <Box
+                className="py-md mt-md"
+                px="xs"
+                mb="md"
+                key={options[0].optionId}
+                sx={(theme) => getQuestionTextStyles(theme, !!options[0].text && options[0].isCorrect)}>
                 <Markdown>{options[0].text}</Markdown>
-              </div>
-            )
-          : options.map((option) => (
-              <Option
-                optionId={option.optionId}
-                checked={selectedChoice === option.optionId}
-                onChange={(value: string) => setSelectedChoice(value)}
-                optionText={option.text}
-                key={option.optionId}
-                className={classNames({
-                  [styles.isAttempted]: isAttempted,
-                  [styles.correct]: option.isCorrect && selectedOptionId === option.optionId,
-                  [styles.inCorrect]: !option.isCorrect && selectedOptionId === option.optionId,
-                  [styles.isPreview]: isPreview,
-                })}
-                disabled={isAttempted}
-              />
-            ))}
-      </div>
-      {/* show reveal answer button for question without options in non preview mode when playing */}
-      {isWithoutOptions && !isPreview && !isAnswerRevealed && (
-        <Button
-          type="button"
-          color="blue"
-          onClick={() => {
-            setIsAnswerRevealed(true);
-
-            // pause timer when the answer is revealed
-            if (pauseTimer) {
-              pauseTimer();
-            }
-          }}>
-          Reveal answer
-        </Button>
-      )}
-      <Divider />
-      {isAttempted && (!isPreview || isQuestionSaved) ? (
-        <Button size="large" onClick={() => onClose()} type="button" color="blue">
-          Close
-        </Button>
-      ) : isWithoutOptions && !isPreview ? (
-        // Show correct & incorrect button in non preview mode when playing
-        // When answer is revealed by quiz host show correct or incorrect button, clicking on which will add points accordingly
-        isAnswerRevealed && (
-          <div className="flex">
-            <Button type="button" size="large" className="fullWidth" color="red" onClick={() => submitResponse(null)}>
-              Incorrect
-            </Button>
+              </Box>
+              {selectedOptionId === '' && (
```

Comment:
What does `selectedOptionId === ''` mean?
Can we declare a variable for this so that it is clear?

Reply:
selectedOptionId === '' means question was not already attempted